geode-core/src/main/java/org/apache/geode/management/internal/configuration/mutators/MemberConfigManager.java

```diff
@@ -59,39 +60,45 @@ public void delete(MemberConfig config, CacheConfig existing) {
 
   @Override
   public List<MemberConfig> list(MemberConfig filter, CacheConfig existing) {
-    String coordinatorId = null;
-    List<MemberConfig> results = new ArrayList<>();
 
-    Set<DistributedMember> members = cache.getDistributionManager().getDistributionManagerIds()
-        .stream().filter(m -> (filter.getId() == null || filter.getId().equals(m.getName())))
-        .map(DistributedMember.class::cast).collect(Collectors.toSet());
+    Set<DistributedMember> distributedMembers =
+        cache.getDistributionManager().getDistributionManagerIds()
+            .stream().filter(internalDistributedMember -> (filter.getId() == null
+                || filter.getId().equals(internalDistributedMember.getName())))
+            .map(DistributedMember.class::cast).collect(Collectors.toSet());
 
-    if (members.size() == 0) {
-      return results;
+    if (distributedMembers.size() == 0) {
+      return new ArrayList<>();
```

Comment:
return `Collections.emptyList()` is preferred to avoid unnecessary allocation

Reply:
Sure we can do that! Thank you!

Reply:
Good point, we'll make the change.